ui/src/common/PlayButton.js

```diff
@@ -30,21 +40,57 @@ export const PlayButton = ({ record, size, className, button }) => {
       })
   }
 
-  return button ? (
-    button()
-  ) : (
-    <IconButton
+  const ButtonType = buttonType === 'button' ? Button : IconButton
+  const translate = useTranslate()
+  const [playing, setPlaying] = useState(false)
```

Comment:
This is the local state of the button which is not in the redux store, that is why `useState` is used. This is set based on `currentTrack`'s `playListAlbumId` and the `recentAlbumOrPlaylist` id.

Reply:
Ah, ok.